docs/configuration/backend.md

```diff
@@ -9,18 +9,25 @@ section to the Ludwig config YAML:
 backend:
   type: ray
   cache_dir: s3://my_bucket/cache
+  cache_credentials: {}
   processor:
     type: dask
   trainer:
     type: horovod
+  loader: {}
 ```
 
 Parameters:
 
 - `type`: How the job will be distributed, one of `local`, `ray`, `horovod`.
 - `cache_dir`: Where the preprocessed data will be written on disk, defaults to the location of the input dataset.
+- `cache_credentials`: Optional dictionary of (or string path to JSON file containing) credentials used to read/write to the `cache_dir`.
 - `processor`: (Ray only) parameters to configure execution of distributed data processing.
 - `trainer`: (Ray only) parameters to configure execution of distributed training.
+- `loader`: (Ray only) parameters to configure loading from preprocessed data into batches.
+
+Note that `cache_credentials` are only needed if your environment does not contain credentials to access the `cache_dir` already. The format
```

Comment:
Nit -- could you move this note to right after where `cache_credentials` are listed, and change it to be part of a callout? Something like:
```
!!! note

    `cache_credentials` are only needed if your environment does not contain credentials to access the `cache_dir`...
```